packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -58,6 +58,10 @@ The Android implementation supports multiple
 [platform view display modes](https://flutter.dev/docs/development/platform-integration/platform-views).
 For details, see [the Android README](https://pub.dev/packages/google_maps_flutter_android#display-mode).
 
+#### Cloud-based map styling
+Cloud-based map styling works on Android platform only if `AndroidMapRenderer.latest` map renderer has been initialized.
```

Comment:
Nit: Blank line above this, matching the rest of the file.

Reply:
Fixed